.subscriptionsrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I would rename it to ctpCredentials

Reply:
For now it is only the credentials, but with time there will be more configs.